analytics/tests/test_views.py

```diff
@@ -698,9 +698,18 @@ def test_scrub_realm(self) -> None:
             self.assert_in_success_response(["Lear &amp; Co. scrubbed"], result)
 
         with mock.patch("analytics.views.do_scrub_realm") as m:
-            with self.assertRaises(AssertionError):
+            with self.assertRaises(AssertionError), self.assertLogs('django.request', 'ERROR') as err_log:
                 result = self.client_post("/activity/support", {"realm_id": f"{lear_realm.id}"})
             m.assert_not_called()
+            self.assertTrue(
+                'ERROR:django.request:Internal Server Error: /activity/support' in err_log.output[0]
+            )
+            self.assertTrue(
+                'Traceback (most recent call last):' in err_log.output[0]
+            )
+            self.assertTrue(
+                'AssertionError' in err_log.output[0]
+            )
```

Comment:
@hackerkid should we replace the AssertionError this is testing with a 40x error?  That seems like a better solution than checking we 500 (generally we should never intentionally 500).